datacenter/ucp/3.0/guides/admin/install/install-on-azure.md

```diff
@@ -0,0 +1,286 @@
+in an ARM template. If you're using stand-alone VMs, you set up IP pools
+for each VM by using a utility container that's configured to run as a
+global Swarm service, which Docker provides.
+
+## Azure Prerequisites 
+
+The following list of infrastructure prerequisites need to be met in order 
+to successfully deploy Docker UCP on Azure.
+
+- All UCP Nodes (Managers and Workers) need to be deployed into the same 
+Azure Resource Group. The Azure Networking (Vnets, Subnets, Security Groups) 
+components could be deployed in a second Azure Resource Group.
+- All UCP Nodes (Managers and Workers) need to be attached to the same 
+Azure Subnet.
+- All UCP (Managers and Workers) need to be tagged in Azure with the 
+`Orchestrator` tag. Note the value for this tag is the Kubernetes version number
+in the format `Orchestrator=Kubernetes:x.y.z`. This value may change in each 
+UCP release. To find the relevant version please see the UCP 
+[Release Notes](../../release-notes). For example for UCP 3.0.6 the tag 
+would be `Orchestrator=Kubernetes:1.8.15`. 
+- The Azure Computer Name needs to match the Node Operating System's Hostname. 
+Note this applies to the FQDN of the host including domain names. 
+- An Azure Service Principal with `Contributor` access to the Azure Resource 
+Group hosting the UCP Nodes. Note, if using a separate networking Resource 
+Group the same Service Principal will need `Network Contributor` access to this 
+Resource Group.
+
+The following information will be required for the installation:
+
+- `subscriptionId` - The Azure Subscription ID in which the UCP 
+objects are being deployed. 
+- `tenantId` - The Azure Active Directory Tenant ID in which the UCP 
+objects are being deployed. 
+- `aadClientId` - The Azure Service Principal ID
+- `aadClientSecret` - The Azure Service Principal Secret Key
+
+### Azure Configuration File
+
+For Docker UCP to integrate in to Microsoft Azure, an Azure configuration file 
+will need to be placed within each UCP node in your cluster. This file 
+will need to be placed at `/etc/kubernetes/azure.json`. 
+
+See the template below. Note entries that do not contain `****` should not be 
+changed.
+
+```
+{
+    "cloud":"AzurePublicCloud", 
+    "tenantId": "***",
+    "subscriptionId": "***",
+    "aadClientId": "***",
+    "aadClientSecret": "***",
+    "resourceGroup": "***",
+    "location": "****",
+    "subnetName": "/****",
+    "securityGroupName": "****",
+    "vnetName": "****",
+    "cloudProviderBackoff": false,
+    "cloudProviderBackoffRetries": 0,
+    "cloudProviderBackoffExponent": 0,
+    "cloudProviderBackoffDuration": 0,
+    "cloudProviderBackoffJitter": 0,
+    "cloudProviderRatelimit": false,
+    "cloudProviderRateLimitQPS": 0,
+    "cloudProviderRateLimitBucket": 0,
+    "useManagedIdentityExtension": false,
+    "useInstanceMetadata": true
+}
+```
+
+There are some optional values for Azure deployments:
```

Comment:
If I did not have an Availability Set what should I set the value to be? `For empty values please set to ""` or similar would be helpful

Reply:
If you don't have an availability set you can leave the value out, hence why i put it in optional :)